ml-dhs-aws/src/test/java/com/marklogic/client/dhs/test/TestJobRunner.java

```diff
@@ -48,17 +70,19 @@ public void testRun() throws IOException {
       String jobFile = JobRunner.jobFileFor(csvFile);
 
       String jobCollection = null;
+      TestJobLogger testJobLogger = new TestJobLogger();
 
       JobRunner jobRunner = new JobRunner();
-      jobRunner.setLogger(logger);
       jobRunner.setRoles(role);
 
       try (
             InputStream csvStream = openStream(csvFile);
             InputStream jobStream = openStream(jobFile);
             InputStream jobStream2 = openStream(jobFile)
       ) {
-         jobRunner.run(client, csvStream, jobStream);
+         jobRunner.run(client, csvStream, jobStream,testJobLogger);
+         assertTrue(testJobLogger.countSuccesses>0);
+         assertTrue(testJobLogger.countFailures<0);
```

Comment:
Would we expect the count of failures to equal 0?  What do you see when you run the unit test?